chore: session replay rn - ga (#10599)
PostHog/posthog.com: contents/docs/session-replay/mobile.mdx

title: Mobile session replay
---

> > 🚧 **NOTE:** Mobile session replay is **currently available on [Android](/docs/session-replay/installation?tab=Android), [iOS](/docs/session-replay/installation?tab=iOS) and [React Native](/docs/session-replay/installation?tab=React%20Native)**. We also have [Flutter SDKs](/docs/session-replay/installation?tab=Flutter) which is in beta (and are free until release). We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow) or one of our other [support options](/docs/support-options).

Author: I think the plural comes from mentioning all SDKs beforehand, but I get your point.
@joethreepwood ?

> > 🚧 **NOTE:** Mobile session replay is **currently available on [Android](/docs/session-replay/installation?tab=Android), [iOS](/docs/session-replay/installation?tab=iOS) and [React Native](/docs/session-replay/installation?tab=React%20Native)**. We also have [Flutter SDKs](/docs/session-replay/installation?tab=Flutter) which is in beta (and are free until release). We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow) or one of our other [support options](/docs/support-options).

Reviewer: ```suggestion
> 🚧 **NOTE:** Mobile session replay is **currently available on [Android](/docs/session-replay/installation?tab=Android), [iOS](/docs/session-replay/installation?tab=iOS) and [React Native](/docs/session-replay/installation?tab=React%20Native)**. We also have a [Flutter SDK](/docs/session-replay/installation?tab=Flutter), which is in beta and free to use until release. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow) or one of our other [support options](/docs/support-options).
```
